src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
 
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final LocalDate from = LocalDate.parse(dateFrom, dateFormatter);
+        final LocalDate to = LocalDate.parse(dateTo, dateFormatter);
```

Comment:
why are they final?

Reply:
I thought it would be good since we have no plans to change these values.

Reply:
ok, was interested if you've done it intentionally, all good)